fast64_internal/oot/c_writer/oot_segment_symbols_c.py

```diff
@@ -62,8 +62,4 @@ def modifySegmentSymbols(scene, exportInfo):
 		segmentSymbols.insert(firstIndex, SegmentSymbolEntry("DECLARE_ROM_SEGMENT", scene.name + "_scene"))
 		firstIndex += 1
 
-		for i in range(len(scene.rooms)):
-			segmentSymbols.insert(firstIndex, SegmentSymbolEntry("DECLARE_ROM_SEGMENT", scene.name + "_room_" + str(i)))
-			firstIndex += 1
```

Comment:
what was this for and why can it be removed now?

Reply:
It adds the segment define to segment_symbols.h, but now it is just extern'd in the c file.